chat-commands.js

```diff
@@ -796,6 +805,7 @@ exports.commands = {
 				return this.errorReply(`This room is already public.`);
 			}
 			if (room.isPersonal) return this.errorReply(`This room can't be made public.`);
+			if (room.parent && room.type === 'chat') return this.errorReply(`Subrooms cannot be public.`);
```

Comment:
I don't see a problem with this?

Honestly, we can make it so subrooms aren't listen in the room list, but they still appear on usercards.

Reply:
>Honestly, we can make it so subrooms aren't listen in the room list, but they still appear on usercards.

That's literally what I'm already doing

Reply:
You should be doing it by marking the subroom as public. :[

Reply:
And then just skipping rooms with a parent in `crq rooms`.